google/resource_compute_attached_disk_test.go

```diff
@@ -0,0 +1,213 @@
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccComputeAttachedDisk_basic(t *testing.T) {
+	t.Parallel()
+
+	diskName := acctest.RandomWithPrefix("tf-test-disk")
+	instanceName := acctest.RandomWithPrefix("tf-test-inst")
+	importID := fmt.Sprintf("%s/us-central1-a/%s:%s", getTestProjectFromEnv(), instanceName, diskName)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: nil,
```

Comment:
You should be able to add an actual CheckDestroy here instead of doing the second TestStep to detach it. If that's not the case, then it would be nice to have a comment below explaining why.